src/main/java/CamWithLensTest.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.Test;
```

Comment:
nur FYI - der test sollte wenn man diese ordner struktur von gradle hat, eigentlich in den /src/test/java ordner - das haben wir noch nicht wirklich gelernt also egal